packages/@lightningjs/ui-components/src/mixins/withThemeStyles/withThemeStyles.test.js

```diff
@@ -0,0 +1,4 @@
+describe('withThemeStyles', () => {
+  beforeEach(() => {});
+  it.skip('should construct with default behavior', () => {});
```

Comment:
is this intentional or does this need to be fleshed out in a future ticket? (or write a todo here?)

Reply:
Future ticket. Added TODO